FaunaDB.Client/Client/StreamingEventHandler.cs

```diff
@@ -2,25 +2,61 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
 using FaunaDB.Errors;
 using FaunaDB.Types;
 
 namespace FaunaDB.Client
 {
     public class StreamingEventHandler : IObservable<Value>, IDisposable
     {
+        private const int TIME_OUT = 10000;
```

Comment:
```suggestion
        private const int TIME_OUT_IN_MILLIS = 10000;
```
or something like that :D